server/api-util/idToken.js

```diff
@@ -79,11 +79,28 @@ exports.openIdConfiguration = (req, res) => {
   });
 };
 
-// Serves the RSA public key as JWK
+/**
+ * @param {String} signingAlg signing algorithm, currently only RS256 is supported
+ * @param {Array} list containing keys to be served in json endpoint
+ *
+ * // Serves the RSA public key as JWK
 // this document is expected to be found from
 // api/.well-known/jwks.json endpoint as stated in discovery document
-exports.jwksUri = (rsaPublicKey, keyId) => (req, res) => {
-  fromKeyLike(crypto.createPublicKey(rsaPublicKey)).then(jwkPublicKey => {
-    res.json({ keys: [{ alg: 'RS256', kid: keyId, use: 'sig', ...jwkPublicKey }] });
+ */
+exports.jwksUri = (signingAlg, keys) => (req, res) => {
```

Comment:
Would it make sense to also include `alg` in the keys that are passed. As the `alg` attribute is key-specific, it would technically allow exposing keys with multiple algorithms. Quite an edge case feature right now but it would then be analogous to the JWK spec as there the algorithm is defined in the key. What do you think?

Reply:
This makes sense!